openmetadata-ui/src/main/resources/ui/src/styles/components/size.less

```diff
@@ -68,9 +71,12 @@
 .w-300 {
   width: 300px;
 }
-.w-60 {
+.w-3\/5 {
   width: 60%;
 }
+.w-60 {
+  width: 15rem;
+}
```

Comment:
have you verified this change? it will impact wherever it is used!

Reply:
Yes, it was not used anywhere else.